.archon/workflows/video/README.md

```
@@ -0,0 +1,114 @@
+archon workflow run ugc-video     "I switched to a standing desk for 30 days"
+archon workflow run product-video "<what it is, who it's for, the one benefit>"
+archon workflow run video-quality-review        # reviews the most recent
+```
+
+## Setup — two keys and ffmpeg
+
+The scripts read these from the **repo-root `.env`** (or Archon-managed
+per-project env vars, which take precedence). They are deliberately not in this
+repo's `.env`, so an unconfigured run fails immediately and by name:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)